docs/superpowers/plans/2026-06-03-token-preferences-dislikes.md

```
@@ -0,0 +1,1299 @@
+  PRIMARY KEY (stake_address, asset_id)
+);
+
+CREATE INDEX IF NOT EXISTS idx_token_preferences_stake
+  ON token_preferences (stake_address, kind, position);
+```
+
+- [ ] **Step 2: Update the wrangler.jsonc comment**
+
+In `wrangler.jsonc`, replace the line
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)